tensordict/prototype/tensorclass.py

```diff
@@ -237,24 +234,26 @@ def wrapper(
 
 
 def _from_tensordict_wrapper(expected_keys):
-    def wrapper(cls, tensordict, non_tensordict=None):
+    def wrapper(cls, td, non_tensordict=None):
         """Tensor class wrapper to instantiate a new tensor class object
 
         Args:
-            tensordict (TensorDict): Dictionary of tensor types
+            td (TensorDict): Dictionary of tensor types
```

Comment:
Instead of renaming the variables throughout, which I think has a minor negative impact on the quality of docstrings and function signatures, what if we alias the import of `tensordict`?